main.py

```diff
@@ -495,7 +505,7 @@ def main(cl_arguments):
             if args.transfer_paradigm == "finetune":
                 # Reload the original best model from before target-task
                 # training since we specifically finetune for each task.
-                pre_target_train = get_best_checkpoint_path(args.run_dir, "pretrain")
+                pre_finetune_path = get_best_checkpoint_path(args.run_dir, "pretrain")
```

Comment:
This'll revert break a fix I recently merged, since the fix makes a change in the opposite direction. Look out when updating.